### PR TITLE
LLM: fixes import from desktop to fully sync accounts before save

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

We introduce a new function `syncNewAccountsToImport` that must be called to provide to `importAccountsReduce` the "syncResult" that result of the synchronisation of all accounts that are about to get imported.

Basically, for user land, we just need to change this:

Screenshot 2022-08-26 at 12 19 53

**This new flow will guarantee that the accounts resulting of a Live QR importation** (importing Accounts from LLD with the animated QR code) will be fully ready to be used: synchronized and complete. 
- This fixes the problem that `tronResources` (or any other coin Resources field) was `undefined` and was causing a lot of random bugs.
- This fixes the problem that the UI was not ready to show the complete accounts (missing transactions, missing secondary balances, bad errors in send flow,...)
- This fixes the ability for us to be truly backward compatible and have a way to deny bad accounts that wouldn't synchronise anymore (e.g. coming from a very old Ledger Live)
- fixes the double import issue ( LIVE-3472 )

#### UX Impact, impact for the end user of LLM

At the end of the importing phase, user will have to wait LLM is synchronising the accounts. This shouldn't take too much time.

All checkbox are replaced by spinners, just to indicate to user that the importing is ongoing. We may figure out better UX in future, notably we do not inform the user for possible sync failure, buggy accounts would simply not get imported (which is an improvement over the past as you could end up with faulty accounts).

### ❓ Context

- **Impacted projects**: LLM 
- **Linked resource(s)**: LIVE-3512 LIVE-3472 LIVE-3510 

### ✅ Checklist

- [x] **Test coverage** 
  - [x] a end to end integration test was added to confirm that imported account throw cross will fully restore their '*Resources' field at the end of the import functions
  - [x] Manually test on Android
  - [ ] Manually test on iOS
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/211411/186947356-b9b2ca67-e50d-43e3-a7d0-0147ac7b2426.mp4

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by haammar-ledger — APPROVED** [review]
LGTM but a review from @LedgerHQ/live-hub would make sense.

File: libs/ledger-live-common/src/account/importAccounts.test.ts [hunk 1]
```diff
@@ -181,13 +181,21 @@ describe("importAccountsMakeItems", () => {
         mode: "update",
       },
     ]);
+    const syncResult = {
+      failed: {},
+      synchronized: {},
+    };
+    result.accounts.forEach((a) => {
+      syncResult.synchronized[a.id] = a;
+    });
```
Comment (alexandremgo) [inline libs/ledger-live-common/src/account/importAccounts.test.ts:190]: the test is defined inside `test.skip("should migrate ethereum accounts", () => ...` (line 15): i think all of it is skipped and never tested

Comment (gre) [reply libs/ledger-live-common/src/account/importAccounts.test.ts:190]: good catch :o

Comment (gre) [reply libs/ledger-live-common/src/account/importAccounts.test.ts:190]: (however, that's not the added test, this was the historical test that I adapted, but i'll try to update it then)

File: apps/ledger-live-mobile/src/screens/ImportAccounts/DisplayResultItem.tsx [hunk 2]
```diff
@@ -38,11 +44,15 @@ export default class DisplayResultItem extends Component<{
         </Flex>
         {!selectable ? null : (
           <Flex ml={8}>
-            <CheckBox
-              onChange={importing ? undefined : this.onSwitch}
-              isChecked={checked}
-              style={styles.marginLeft}
-            />
+            {importing ? (
+              <Importing />
```
Comment (LFBarreto) [inline apps/ledger-live-mobile/src/screens/ImportAccounts/DisplayResultItem.tsx:48]: I would suggest importing InfiniteLoader from native-ui lib instead
```js
import { InfiniteLoader } from "@ledgerhq/native-ui";

...

<InfiniteLoader />
```

**Review by LFBarreto — CHANGES_REQUESTED** [review]

**Comment by gre** [comment]
https://github.com/LedgerHQ/ledger-live/actions/runs/2946858867

File: apps/ledger-live-mobile/src/screens/ImportAccounts/DisplayResultItem.tsx (hunk 2 above)
Comment (gre) [reply apps/ledger-live-mobile/src/screens/ImportAccounts/DisplayResultItem.tsx:48]: ok, i'm updating the code, but i can't test it locally (my computer have issue building Android :'( )

**Comment by gre** [comment]
@LFBarreto

**Comment by LFBarreto** [comment]
>  @LFBarreto

probably a smaller size loader then  
```js
<InfiniteLoader size={20} />
```
Looks a little big imo :sweat_smile:

**Review by LFBarreto — APPROVED** [review]

**Comment by gre** [comment]
let's iterate on this properly in future. for now we need this hotfix in 🙏  the multi spinner idea is probably not the best in any case, we can tweak later

**Pull request merged**